### DISCUSSION TITLE
Does one have to use the package directory to link a workspace?

### DISCUSSION THREAD

**Discussion by salieflewis** [pnpm/pnpm#6810] [General]
Lets say I just have one package that I want to consume inside my monorepo, can I just link to that package inside my yaml file instead of creating a packages directory? So my yaml would look like this:

```
# pnpm-workspace.yaml
packages:
  - 'my-package'
```

Instead of:

```
# pnpm-workspace.yaml
packages:
  - 'packages/*'
```

**Comment by zhangzhuang15** [comment]
"packages" directory is not necessary, many people use it because they prefer this name style.As long as your "my-package" is a directory, your yaml config is ok.
